docusaurus/docs/learn/quickstarts/network/help/quickstart-walkthrough.md

```diff
@@ -0,0 +1,314 @@
+---
+title: Quickstart Walkthrough
+id: quickstart-walkthrough
+---
+The following will walk you through the manual process of creating an overlay network with your own PKI. These steps are the same steps that take place behind the scenes when running the OpenZiti Quickstart’s `expressInstall` function.
```

Comment:
> The following will

feels like it's missing "document", or "page", or "article" or "something"?

> the manual process of creating an overlay network

Maybe change this paragraph to:

> This page is intended to explain the steps that happen automatically when the `expressInstall` function is executed. The [Local - No Docker](add link), [Local - With Docker](add link), [Local - Docker Compose](add link), and [Host OpenZiti Anywhere] quickstarts all run the `expressInstall` function. Each version varies slightly. This page will focus on the [Host OpenZiti Anywhere](link) quickstart.

Reply:
It's probably best to 'outline' what happens here too.
* make directory
* establish env vars
* create config for controller
* init ziti controller
* create router config
* create router in controller
* enroll router
* add default policies (erp/serp)

That basically gives you the rest of the outline to then fill in where you can add things like what each env var is, and which ones are important, what the pki looks like, why it looks like it does (aka 'complex' vs 'as simple as necessary') etc.